tests/lib/rules/key-spacing.js

```diff
@@ -1058,5 +1058,217 @@ ruleTester.run("key-spacing", rule, {
             { message: "Missing space before value for key 'a'.", line: 1, column: 6, type: "Identifier" },
             { message: "Extra space after key 'c'.", line: 1, column: 20, type: "Identifier" }
         ]
+    },
+
+    // https://github.com/eslint/eslint/issues/5613
+    {
+        code: [
+            "({",
+            "    longName : 1,",
+            "    small :    2,",
+            "    xs :       3",
+            "})"
+        ].join("\n"),
+        options: [{
+            multiLine: {
+                align: {
+                    on: "value",
+                    beforeColon: true,
+                    afterColon: true
+                }
+            }
+        }],
+        parserOptions: { ecmaVersion: 6, ecmaFeatures: { experimentalObjectRestSpread: true } },
+        errors: []
```

Comment:
For all the tests that have no errors, there's a separate array of `valid` tests above.